doc/release-notes/release-notes-16152.md

```diff
@@ -0,0 +1,3 @@
+Miscellaneous CLI Changes
```

Comment:
This is not a CLI change, but a change of default p2p behavior. (You remove the `NODE_BLOOM` service bit by default)

Reply:
I dont know what the list of categories are, but I think a default parameter change is a CLI change. Happy to change to whatever.